acceptance-tests/src/test/java/tech/pegasys/signers/dsl/hashicorp/HashicorpNode.java

```diff
@@ -94,22 +81,10 @@ public int getPort() {
   }
 
   public boolean isTlsEnabled() {
-    return tlsServerCertificate != null;
-  }
-
-  public Optional<Path> getKnownServerFilePath() {
-    return knownServerFile;
+    return serverTlsCertificate != null;
```

Comment:
Shouldn't this just be serverTlsCertifcate.isPresent considering it's an optional